packages/columns/src/Column.stories.tsx

```diff
@@ -202,3 +205,63 @@ RowAndColumnGaps.args = {
 	rowGap: 2,
 	columnGap: 0.5,
 };
+
+export const ThreeColumn: ComponentStory<typeof Columns> = (args) => (
```

Comment:
Having a story where some props are configurable is a bit weird... I wonder if it's easier to make this story non-configurable? Or move the `cols` props to args